ysoserial/Generators/WindowsPrincipalGenerator.cs

```diff
@@ -79,9 +81,9 @@ public override object Generate(string formatter, InputArgs inputArgs)
             else if (formatter.ToLower().Equals("json.net"))
             {
                 string payload = @"{
-                    '$type': 'System.Security.Principal.WindowsPrincipal, mscorlib',
+                    '$type': 'System.Security.Principal.WindowsPrincipal',
```

Comment:
Please do not shorten it by hand - the minifier should take care of this (we need to keep it useful for researchers when reading it)